app/src/main/java/org/simple/clinic/bp/assignbppassport/BpPassportSheet.kt

```diff
@@ -0,0 +1,13 @@
+package org.simple.clinic.bp.assignbppassport
+
+import android.os.Bundle
+import org.simple.clinic.R
+import org.simple.clinic.widgets.BottomSheetActivity
+
+class BpPassportSheet : BottomSheetActivity() {
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.sheet_bp_passport)
```

Comment:
We should be using Viewbinding.

Reply:
ViewBinding is used in the later commits